cli.js

```diff
@@ -106,6 +106,11 @@ function init(files) {
 			return path.join(process.cwd(), file);
 		})
 		.then(function (files) {
+			if (files.length === 0) {
```

Comment:
@sindresorhus don't we go for `!files.length`? Or am I just thinking back to my C days?